src/vsgi-application.vala

```diff
@@ -76,57 +75,6 @@ public class VSGI.Application : GLib.Application {
 				case "default":
 					Log.set_writer_func ((LogWriterFunc) Log.writer_default);
 					break;
-#else
-				case "standard-streams":
-					if (Posix.isatty (stderr.fileno ())) {
-						Log.set_default_handler ((domain, level, message) => {
-							stderr.printf ("[%s] %s%s:%s %s%s%s%s\n",
-							               new DateTime.now_utc ().format ("%FT%H:%M:%S.000Z"),
-							               "\x1b[33m",
-							               "worker %d".printf (Posix.getpid ()),
-							               "\x1b[0m",
-							               domain == null ? "" : "%s: ".printf (domain),
-							               LogLevelFlags.LEVEL_ERROR    in level ? "\x1b[31m" :
-							               LogLevelFlags.LEVEL_CRITICAL in level ? "\x1b[31m" :
-							               LogLevelFlags.LEVEL_WARNING  in level ? "\x1b[33m" :
-							               LogLevelFlags.LEVEL_MESSAGE  in level ? "\x1b[32m" :
-							               LogLevelFlags.LEVEL_INFO     in level ? "\x1b[34m" :
-							               LogLevelFlags.LEVEL_DEBUG    in level ? "\x1b[36m" : "",
-							               message.replace ("\n", "\n\t\t"),
-							               "\x1b[0m");
-						});
-					} else {
-						Log.set_default_handler ((domain, level, message) => {
-							Log.default_handler (domain, level, "[%s] %s".printf (new DateTime.now_utc ().format ("%FT%H:%M:%S.000Z"), message));
-						});
-					}
-					break;
-#if LIBSYSTEMD
```

Comment:
I see, we only use `LIBSYSTEMD` when not natively supported by GLib. In that case, you should also remove `libsystemd.vapi` since it's not needed anymore.